packages/patternfly-4/react-core/src/components/AboutModal/examples/SimpleAboutModal.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { AboutModal, Button } from '@patternfly/react-core';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/components/Content/styles.css';
```

Comment:
I think we don't want consumers to do this?

Reply:
I agree.  This was the only way I could find that worked  to apply the styles that  do not have a component.  However, this dopes in work in live mode docs.